ios-theater/View/BannerCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  BannerCell.swift
+//  ios-theater
+//
+//  Created by Linxiao Wei on 2019/12/18.
+//  Copyright © 2019 Linxiao Wei. All rights reserved.
+//
+
+import UIKit
+
+class BannerCell: UICollectionViewCell {
+  @IBOutlet weak var imageView: UIImageView!
+  
+  func configure(image: String) {
```

Comment:
How about renaming the parameter to `imageName` or `name`?

Reply:
Okay, I'll rename the parameter to `imageName`